src/main/java/com/cloudbees/jenkins/GitHubPushTrigger.java

```diff
@@ -114,7 +118,12 @@ public void run() {
                         LOGGER.log(Level.WARNING, "Failed to parse the polling log",e);
                         cause = new GitHubPushCause(pushBy);
                     }
-                    if (job.scheduleBuild(cause)) {
+                    ParameterizedJobMixIn scheduler = new ParameterizedJobMixIn() {
```

Comment:
@daniel-beck ```scheduledJob``` works for me. I'll change it.
@KostyaSha The ```job``` variable is an instance of ```Job``` which doesn't have any of the ```schedule*``` methods. If you know a better way to schedule a build based on a ```Job``` instance, please tell me, I'll be happy to modify it.